guides/v2.0/config-guide/bootstrap/mage-profiler.md

```diff
@@ -22,7 +22,7 @@ Magento profiling enables you to:
 
 	You should be particularly interested in the list of *unused dependencies*, which are objects that were created because they were requested in some constructor, but were never used (that is, none of their methods were called). As a result, processor time and memory spent to create these dependencies are wasted.
 
-Magento provides the base functionality in <a href="{{ site.mage2000url }}lib/internal/Magento/Framework/Profiler.php" target="_blank">Magento\Framework\Profiler</a>.
+Magento provides the base functionality in [Magento\\Framework\\Profiler]({{ site.mage2000url }}lib/internal/Magento/Framework/Profiler.php){: target="_blank"}.
```

Comment:
Not supposed to be two `\` and I don't think there is supposed to be a space between `:` and `target` here.